### PR TITLE
feat: add now autodeploy

### PR DESCRIPTION
**Pull request by nikolaevigor**
ref: #24 

I've added auto-deployment, could someone review, please?

`NOW_TOKEN` should be set as environment variable in travis.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,5 +70,8 @@
     "snapshotSerializers": [
       "enzyme-to-json/serializer"
     ]
+  },
+  "now": {
+    "alias": "styled-components-website"
```
Comment (mxstbr) [inline package.json]: This should be `styled-components.com`, right?

**Comment by mxstbr** [comment]
Unfortunately Zeit doesn't have API tokens for teams yet so we can't actually do this I don't think :confused:

File: package.json (hunk 1 above)
Comment (nikolaevigor) [reply package.json]: Not sure. In my last project final url generated by Now was `<alias>.now.sh`

**Comment by nikolaevigor** [comment]
According to this [issue](https://github.com/zeit/now-cli/issues/655), it seems, that we can deploy like this:
`yarn build && now switch $TEAM_NAME –token=$NOW_TOKEN && now build --public --token $NOW_TOKEN --name styled-components-website && now alias --token $NOW_TOKEN –force`
Also related: [link1](https://github.com/zeit/now-cli/issues/649), [link2](https://github.com/zeit/now-cli/issues/676)

**Comment by mxstbr** [comment]
Oh no wtf did I do here :scream: Oh god I messed it all up

**Comment by mxstbr** [comment]
@nikolaevigor sorry about that, would you mind force pushing this branch again? I totally messed this up :confused:

```sh
git push origin feat/travis-now-autodeploy --force
```

**Comment by morajabi** [comment]
oh, my, god! Never saw something like this before 😅

**Comment by mxstbr** [comment]
Wait now it's fixed again? :scream:

**Comment by nikolaevigor** [comment]
@mxstbr it's me :)

**Comment by probablyup** [comment]
Let's do this!! @mxstbr is this good to merge or is more setup needed?

**Comment by probablyup** [comment]
Ended up going a different direction for this, but the functionality is now up and working!

**Pull request closed without merging**